conan/api/subapi/cache.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, conan_api):
 
     def export_path(self, ref: RecipeReference):
         cache = PkgCache(self.conan_api.cache_folder, self.conan_api.config.global_conf)
-        ref.revision = None if ref.revision == "latest" else ref.revision
+        ref = _resolve_latest_ref(cache, ref)
```

Comment:
To make it uniform, some methods were using ``_resolve_latest_pref()`` below, and some were doing it inline.